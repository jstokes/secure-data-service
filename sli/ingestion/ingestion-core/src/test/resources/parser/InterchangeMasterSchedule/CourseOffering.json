{
  "@id":"ID000",
  "LocalCourseCode":
  {
    "_value":"LocalCourseCode0"
  },
  "LocalCourseTitle":
  {
    "_value":"LocalCourseTitle0"
  },
  "SchoolReference":
  {
    "EducationalOrgIdentity":
    {
      "StateOrganizationId":
      {
        "_value":"StateOrganizationId0"
      }
    }
  },
  "SessionReference":
  {
    "SessionIdentity":
    {
      "EducationalOrgReference":
      {
        "EducationalOrgIdentity":
        {
          "StateOrganizationId":
          {
            "_value":"StateOrganizationId1"
          }
        }
      },
      "SessionName":
      {
        "_value":"SessionName0"
      }
    }
  },
  "CourseReference":
  {
    "CourseIdentity":
    {
      "EducationalOrgReference":
      {
        "EducationalOrgIdentity":
        {
          "StateOrganizationId":
          {
            "_value":"StateOrganizationId2"
          }
        }
      },
      "UniqueCourseId":
      {
        "_value":"UniqueCourseId0"
      }
    }
  }
}
