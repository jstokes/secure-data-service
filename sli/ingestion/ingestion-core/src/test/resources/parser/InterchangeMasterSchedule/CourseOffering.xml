<?xml version="1.0" encoding="UTF-8"?>
<InterchangeMasterSchedule xmlns:sli="http://slc-sli/ed-org/0.1"
 xmlns:ns0="http://ed-fi.org/0100"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 xsi:schemaLocation="http://ed-fi.org/0100 file:/C:/Users/tshewchuk/repository/sli/edfi-schema/src/main/resources/edfiXsd-SLI/SLI-Interchange-MasterSchedule.xsd">
    <CourseOffering id="ID000">
        <LocalCourseCode>LocalCourseCode0</LocalCourseCode>
        <LocalCourseTitle>LocalCourseTitle0</LocalCourseTitle>
        <SchoolReference>
            <EducationalOrgIdentity>
                <StateOrganizationId>StateOrganizationId0</StateOrganizationId>
            </EducationalOrgIdentity>
        </SchoolReference>
        <SessionReference>
            <SessionIdentity>
                <EducationalOrgReference>
                    <EducationalOrgIdentity>
                        <StateOrganizationId>StateOrganizationId1</StateOrganizationId>
                    </EducationalOrgIdentity>
                </EducationalOrgReference>
                <SessionName>SessionName0</SessionName>
            </SessionIdentity>
        </SessionReference>
        <CourseReference>
            <CourseIdentity>
                <EducationalOrgReference>
                    <EducationalOrgIdentity>
                        <StateOrganizationId>StateOrganizationId2</StateOrganizationId>
                    </EducationalOrgIdentity>
                </EducationalOrgReference>
                <UniqueCourseId>UniqueCourseId0</UniqueCourseId>
            </CourseIdentity>
        </CourseReference>
    </CourseOffering>
</InterchangeMasterSchedule>
