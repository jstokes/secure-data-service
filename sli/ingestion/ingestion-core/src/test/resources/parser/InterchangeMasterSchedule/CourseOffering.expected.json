{
  "@id":"ID001",
  "LocalCourseCode":{"_value":"eiKnCG"},
  "LocalCourseTitle":{"_value":"w4w6GbeZ8u"},
  "SchoolReference":{
    "EducationalOrgIdentity":{
      "StateOrganizationId":{"_value":"qU8Hus"}
    }
  },
  "SessionReference":{
    "SessionIdentity":{
      "EducationalOrgReference":{
        "EducationalOrgIdentity":{
          "StateOrganizationId":{"_value":"AEKtTs2zMP10B5"}
        }
      },
      "SessionName":{"_value":"txtx1h13V"}
    }
  },
  "CourseReference":{
    "CourseIdentity":{
      "EducationalOrgReference":{
        "EducationalOrgIdentity":{
          "StateOrganizationId":{"_value":"KK.6_D97.8TaAQSPw"}
        }
      },
      "UniqueCourseId":{"_value":"Q7wc2O4-vBul-pnuf5s"}
    }
  }
}